test/TEST-16-ISSUE-518/avocado.conf

```diff
@@ -0,0 +1,2 @@
+[datadir.paths]
+logs_dir = /tmp/job-results
```

Comment:
I'd really stick to the framework we already have instead of adding new infrastructure just for one test...